docs/user-guide/generate-certificates.md

```diff
@@ -14,11 +15,11 @@ Follow these steps to generate a PKCS12 keystore:
 1. [Configure the PKCS12 setup section in zowe.yaml](#configure-the-pkcs12-setup-section-in-zoweyaml)
 2. [Run the command to generate a PKCS12 keystore](#run-the-command-to-generate-a-pkcs12-keystore)
 
-### Configure the PKCS12 setup section in zowe.yaml
+### Configuring the PKCS12 setup section in zowe.yaml
```

Comment:
*in Zowe configuration YAML file

Reply:
important point ^^^
zowe configuration can be in parmlib, so we should not say 'zowe.yaml' anywhere really.